packages/react-components/src/molecules/CtaCard.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import css from '@emotion/css';
+
+import { Link, Card, Paragraph } from '../atoms';
+import {
+  mobileScreen,
+  perRem,
+  vminLinearCalcClamped,
+  tabletScreen,
+  smallDesktopScreen,
+} from '../pixels';
+
+const getInTouchStyles = css({
+  display: 'grid',
+  gridTemplateColumns: 'auto',
+  alignItems: 'center',
+  paddingTop: `${12 / perRem}em`,
+  paddingLeft: `${vminLinearCalcClamped(
+    mobileScreen,
+    24,
+    tabletScreen,
+    36,
+    'px',
+  )}`,
+  paddingRight: `${vminLinearCalcClamped(
+    mobileScreen,
+    24,
+    tabletScreen,
+    36,
+    'px',
+  )}`,
+  paddingBottom: `${vminLinearCalcClamped(
+    mobileScreen,
+    18,
+    tabletScreen,
+    24,
+    'px',
+  )}`,
+  [`@media (min-width: ${smallDesktopScreen.min}px)`]: {
+    gridTemplateColumns: 'auto min-content',
+  },
+});
+
+const buttonStyles = css({
+  flexGrow: 1,
+  display: 'flex',
+  justifyContent: 'center',
+  [`@media (min-width: ${tabletScreen.min}px)`]: {
+    display: 'block',
+  },
+});
+
+type CtaCardProps = {
+  href: string;
```

Comment:
`children: TextChildren` to restrict and because children should probably be mandatory for this component

Reply:
oh nvm, not text children since we need more in it, but still mandatory